_data/current.yml

```diff
@@ -1,3 +1,10 @@
+
+- name: "Testit2022"
+  location: Carlsgatan 10C, 211 20 Malmö, Sweden
+  dates: "September 13, 2022"
```

Comment:
Please move this to the correct placement. It needs to be in the correct date order.

Reply:
Thank you. Placing it in the right location now.